Slim/Plugin/ExtendedBrowseModes/Settings.pm

```diff
@@ -107,6 +107,10 @@ sub handler {
 					$menu->{id}     = 'myMusicAlbums' . $menu->{id} if $menu->{id} !~ /^myMusic/;
 					$menu->{weight} = "25.$ts" * 1;
 				}
+				if ( $feedType eq 'works' ) {
```

Comment:
A really minor thing: could you please make this an `elsif`? Thanks!

Reply:
I read your mind!

Incidentally, it's this that makes it magically work with Material.